src/components/App.jsx

```diff
@@ -148,8 +148,11 @@ export default class App extends React.Component {
 
     const styleUrl = initialStyleUrl()
     if(styleUrl) {
```

Comment:
@pathmapper currently when you press cancel it doesn't load the users previously loaded style. I think you want

```
if(styleUrl || window.confirm(...)) {
```

So it falls into the `else`block when the user presses cancel.

Also how about adding the URL being loaded to the confirm message.

Reply:
Yeah, good point, thanks! If canceled, it falls now in the `else` block. If present, the `style= `parameter is now also removed there.

Also added the the URL to the confirm message, good idea.